nni/retiarii/oneshot/pytorch/supermodule/operation.py

```diff
@@ -472,6 +473,44 @@ def forward_with_args(self,
             eps,
         )
 
+class MixedLayerNorm(MixedOperation, nn.LayerNorm):
+    """
+    Mixed LayerNorm operation.
```

Comment:
Missing a lot of blank lines here. Please check the syntax of RST, as well as [numpydoc](https://numpydoc.readthedocs.io/en/latest/format.html).